yarn-project/circuit-types/src/logs/l1_note_payload/encrypt_buffer.ts

```diff
@@ -59,12 +57,11 @@ export function encryptBuffer(
  *
  * @param data - The encrypted data buffer to be decrypted.
  * @param ownerPrivKey - The private key of the owner used for decryption.
- * @param curve - The curve object used in the decryption process.
  * @returns The decrypted plaintext as a Buffer or undefined if decryption fails.
  */
-export function decryptBuffer(data: Buffer, ownerPrivKey: GrumpkinPrivateKey, curve: Grumpkin): Buffer | undefined {
+export function decryptBuffer(data: Buffer, ownerPrivKey: GrumpkinPrivateKey): Buffer | undefined {
```

Comment:
I think we will be running into some issues when we get to the outgoing as it is passing different things in to decrypt etc 🤷 Potentially it could be useful for us to have fully separate functions for it, and only share the small pieces to not have confusion when we get to that.